src/views/TimeOutWrapper/index.tsx

```diff
@@ -18,6 +20,10 @@ const TimeOutWrapper = ({ children }: TimeOutWrapperProps) => {
     Duration.fromObject({ minutes: 5 }).as('seconds')
   );
   const LAST_ACTIVE_AT_KEY = 'easiLastActiveAt';
+  const isLocalAuth =
+    isLocalAuthEnabled() &&
+    window.localStorage[localAuthStorageKey] &&
+    JSON.parse(window.localStorage[localAuthStorageKey]).favorLocalAuth;
```

Comment:
I know we do this in other places, but what does checking the value of `favorLocalAuth` do that the line above doesn't? Is there a case where `window.localStorage[localAuthStorageKey]` would be truthy but we'd still not consider a user to be logged in via local auth?